sdk/eventhubs/azure-messaging-eventhubs-track1-perf/pom.xml

```diff
@@ -30,6 +30,11 @@
       <artifactId>perf-test-core</artifactId>
       <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:perf-test-core;current} -->
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
```

Comment:
Adding `jackson-core` was sufficient in removing that class not found exception.  Unless you are doing something different.

https://github.com/Azure/azure-sdk-for-java/pull/33375/files#diff-e70fa89b63014fa4fe7dbd361afaf9f7bbd80c41d2b59ce0ed5c012379cfb46fR34

Reply:
Updating it in next commit.